MVC-App/Controllers/RelationsController.cs

```diff
@@ -44,26 +45,100 @@ private async Task<List<RelationViewModel>> InitRelationModels()
                                 FullName = tblRelation.FullName,
                                 TelephoneNumber = tblRelation.TelephoneNumber,
                                 Email = tblRelation.EMailAddress,
-                                Country = tblRelationAddress.CountryName,
+                                CountryId = tblCountry.Id,
+                                CountryName = tblCountry.Name,
                                 City = tblRelationAddress.City,
                                 Street = tblRelationAddress.Street,
                                 PostalCode = tblRelationAddress.PostalCode,
+                                PostalCodeMask = tblCountry.PostalCodeFormat,
                                 StreetNumber = tblRelationAddress.Number ?? 0
                             };
             return await relationModels.ToListAsync();
         }
 
+        private string ApplyMask(string value, string mask)
```

Comment:
Please create a MVC-App.Tests project inside your solution and test method ApplyMask() to satisfy that it work correctly.

Reply:
You can choose MSUnit as a framework. Test different cases providing expected value.